pkg/trace/obfuscate/obfuscate.go

```diff
@@ -38,8 +38,8 @@ type Obfuscator struct {
 // SQLOptions holds options that change the behavior of the obfuscator for SQL.
 // easyjson:json
 type SQLOptions struct {
-	// QuantizeSQLTables determines if the obfuscator will perform quantization on the SQL tables.
-	QuantizeSQLTables bool `json:"quantize_sql_tables"`
+	// QuantizeSQL determines if the obfuscator will perform quantization on the SQL statement.
+	QuantizeSQL bool `json:"quantize_sql"`
```

Comment:
This is not clear enough. Obfuscation in itself is already quantization. When we read `QuantizeSQL` it doesn't say much, nor the documentation. In fact, what the option that you have here does is that it determines the quantization of digits in identifiers. Please call this option `ReplaceDigits` or `QuantizeIDs` (no need for the word "sql" since the containing structure is already called "SQLOptions")

Reply:
Great suggestion, updated: https://github.com/DataDog/datadog-agent/pull/8860/commits/83f3c03ef054618f7026747466d60e764694c041